crates/autopilot/src/boundary/mod.rs

```diff
@@ -43,3 +43,9 @@ pub fn buffered_web3_client(ethrpc: &Url) -> Web3 {
         });
     shared::ethrpc::web3(&ethrpc_args, &http_factory, ethrpc, "base")
 }
+
+pub struct SolvableOrders {
+    pub orders: Vec<model::order::Order>,
+    pub quotes: HashMap<crate::domain::OrderUid, Quote>,
+    pub latest_settlement_block: u64,
```

Comment:
nit: would be nice to have `eth::BlockNo` here instead of `u64`, but we add it later when we introduce `eth` module.

Reply:
We don't have `eth` in dependencies for this crate, also `Auction` also uses `u64`. https://github.com/cowprotocol/services/blob/further-order-event-relief/crates/autopilot/src/domain/auction/mod.rs#L13